<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseClass</class>
 <widget class="QWidget" name="ChooseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>筛选</string>
  </property>
  <widget class="QRadioButton" name="all_rbtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>随机背诵全部单词</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="select_rbtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按条件筛选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>开始背诵</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="tag_cb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按标签筛选</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="zk_cb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>中考</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="gk_cb">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>高考</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cet4_cb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CET4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="collins3_rbtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="collins_cb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>柯林斯星级</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cet6_cb">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CET6</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="collins1_rbtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="collins2_rbtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="oxford_cb">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>130</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>牛津核心3000词</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>220</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>619</x>
     <y>50</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="bnc_cb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>英国国家语料库词频顺序</string>
   </property>
  </widget>
  <widget class="QSlider" name="bnc_slider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="bnc_tl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="frq_slider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="frq_tl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="frq_cb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>170</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当代语料库词频顺序</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wordcount_tl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="pos_cb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>490</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按词性筛选</string>
   </property>
  </widget>
  <widget class="QLabel" name="info0">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>前</string>
   </property>
  </widget>
  <widget class="QLabel" name="info1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>前</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="prep_cb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>介词</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="adv_cb">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>副词</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="verb_cb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>520</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>动词</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="n_cb">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>520</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名词</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="adj_cb">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>520</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>形容词</string>
   </property>
  </widget>
  <widget class="QLabel" name="info2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>最多:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="collins4_rbtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="collins5_rbtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="posothers_cb">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>520</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>其它</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ky_cb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>考研</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ielts_cb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>雅思</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="toefl_cb">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>托福</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
